@kiva/kv-components/vue/KvLightbox.vue

```diff
@@ -263,6 +264,8 @@ export default {
 
 		const onScreenClick = () => {
 			if (!preventClose.value) {
+				// This emit is only for MARS-199 experiment tracking and should be removed after results
+				emit('lightbox-closed-click-outside');
```

Comment:
Is this really necessary? Can't we just track both close events with the same se_action param? Also, I'm pretty sure this will also fire the close event `emit('lightbox-closed');`

Reply:
If we wanted to keep just a single `lightbox-closed` event, which I do prefer, we could pass a payload along with the event and ensure each call to `hide(type)` includes the type.

then the emit would look something like `emit('lightbox-closed', { type: "button/close-x/background-click" })`

Reply:
I prefer that too! will work on that to make it cleaner